### PR TITLE
{data}[foss/2023a] scib v1.1.4, pyparsing v3.1.1, pydot v2.0.0, Deprecated 1.2.14, scikit-misc v0.3.1

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/p/pyparsing/pyparsing-3.1.1-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+executing simple grammars, vs. the traditional lex/yacc approach, or the use of
+regular expressions. The pyparsing module provides a library of classes that
+client code uses to construct the grammar directly in Python code."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+# matplotlib 3.7.2 requires pyparsing<=3.1, this v3.1.1 will not works with it
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkg-config', '0.29.2', '', SYSTEM),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/pyparsing/pyparsing-3.1.1-GCCcore-12.3.0.eb:17]: `pkg-config` is no longer actively maintained, so please use `pkgconf` instead
```suggestion
    ('pkgconf', '1.9.5'),
```

**Comment by pavelToman** [comment]
Waiting for 
- https://github.com/easybuilders/easybuild-easyconfigs/pull/19981

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/27f33641fabce0bcac686bef4d6b0374 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**